src/assets/index.css

```diff
@@ -1,3 +1,58 @@
+@font-face {
+    font-family: 'Raleway';
+    src: url('https://rawgit.com/voorhoede/voorhoede-ocelot-formatter/fonts/src/assets/fonts/raleway-extrabold-webfont.woff2') format("woff2"),
```

Comment:
I would use local font notation in source code and use a build task to replace with remote url during deploy.

So in source, something like:
```css
@font-face {
  font-family: 'Raleway';
  src: url('assets/fonts/raleway-extrabold-webfont.woff2') format("woff2"),
         url('assets/fonts/raleway-extrabold-webfont.woff') format("woff");
  font-weight: 700;
  font-style: normal;
}
```
And in build replace `assets/fonts/` with `https://rawgit.com/voorhoede/voorhoede-ocelot-formatter/fonts/src/assets/fonts/` or `https://voorhoede.github.io/voorhoede-ocelot-formatter/assets/fonts/`.